lib/components/Masthead/Masthead.module.scss

```diff
@@ -104,6 +147,23 @@ $branding-min-width-size: 120px;
             justify-content: flex-end;
             list-style-type: none;
 
+            .masthead-toolbar-menu ul {
+                list-style: none;
+                padding-left: 0;
+                width: $layout-nav-expanded-width;
+
+                li {

```

Comment:
add styling through classes? We already define masthead-toolbar-btn

Reply:
added a specific class 'masthead-toolbar-btn-container'. This is one level up to masthead-toolbar-btn